.github/workflows/inspektor-gadget.yml

```diff
@@ -28,9 +28,9 @@ jobs:
       id: publish-registry
       uses: elgohr/Publish-Docker-Github-Action@2.8
       with:
-        name: docker.io/kinvolk/gadget
+        name: ${{ secrets.DOCKER_REPO }}
         username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
`DOCKER_USERNAME` -> `CONTAINER_REGISTRY_USERNAME`

(and other occurrences below)

Reply:
Yes, something went wrong during a rebase.